### ISSUE TITLE
Bug: App crashes on startup

### ISSUE THREAD

**Issue by HelderRocket** [0x5bfa/FluentHub#351]
### Before you start...

- [X] Have you updated FluentHub? You might be using an old version.
- [X] Have you checked if a similar issue has already been reported?

### What version of Windows is this issue present?

Windows 10

### What version/architecture of FluentHub are you on?

0.5.0.0 (The ap does not let me enter the information of the same).

### Description

Every time I try to interact with the application, looking for a repository or a profile (or even just by going to the page of, for example "favorites"), it crashes abruptly, with no error message.

### Steps To Reproduce

I click on any on-screen option in the main menu.

### Expected behavior

I don't know how the application works, but I guess it's some code that is not connecting to the source.

### Stack Trace

N/A

### Assets

In the following link, you can see how the error is reproduced (the video is too big to put it here directly):
https://drive.google.com/file/d/1uYPUlYc4GgjKsQTdtTyf2iCfhLyMWjUu/view?usp=share_link

**Comment by onein528** [comment]
I looked that video and it seems to be repository-related code. I will reproduce that with clicking the same repo as you.

**Comment by onein528** [comment]
@HelderRocket Can you send log? You can find it in 'C:¥Users¥USER_NAME¥AppData¥Local¥Packages¥~~fluenthub~~'

**Comment by onein528** [comment]
I know the reason why you cannot access profile page, repo page, issue page, PR page, and like so. That's due to WebView2 issues.

**Comment by HelderRocket** [comment]
> @HelderRocket Can you send log? You can find it in 'C:¥Users¥USER_NAME¥AppData¥Local¥Packages¥~fluenthub~'

[Log20221025.log](https://github.com/FluentHub/FluentHub/files/10708044/Log20221025.log)
[Log20221126.log](https://github.com/FluentHub/FluentHub/files/10708047/Log20221126.log)
[Log20221012.log](https://github.com/FluentHub/FluentHub/files/10708048/Log20221012.log)
[Log20221016.log](https://github.com/FluentHub/FluentHub/files/10708050/Log20221016.log)
[Log20221020.log](https://github.com/FluentHub/FluentHub/files/10708052/Log20221020.log)

**Comment by HelderRocket** [comment]
I am surprised that the ".log" files are only from last year, none from this year.

**Comment by 0x5bfa** [comment]
I removed WebView2 components in the latest commit.

If you are able to build the source, you can see what's changed in the next release.

v0.5.1 shipment is gonna be in a week.

**Comment by 0x5bfa** [comment]
I uploaded the whole new shipment into the Store. Can you try if it is available already? Version: 0.8.9

**Comment by HelderRocket** [comment]
Yes, it is now available. Although the error is still present. At least when checking the repositories you gave a star to. But it no longer happens when entering a repository or on the main page (unless you use a screen recorder).
![image](https://github.com/FluentHub/FluentHub/assets/104467221/f83a50d7-9ea9-4eed-8829-5ba5aff5f640)

**Comment by 0x5bfa** [comment]
Almost all of crashes should be fixed and displayed exception message on the page. can you try 0.8.40 and see the error message?
